index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mobile Set Up</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./index.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="top-nav">
+        <p>John Doe</p>
+        <img src="./Assets/Menu.svg" alt="humburger menu" />
+      </nav>
+    </header>
+    <main>
+      <section>
+        <header>
+          <h1 class="introduction">Hey There.<br />I'm Linda</h1>
+          <h3 class="profession">I'am a software Developer</h3>
+        </header>
+        <p class="content">
+          I can help you build a product,Feature or website.Lookthrough some of
+          my work and experience !If you like what you see and have a project
+          you need coded,<br />dont hesistate to conduct me.
+        </p>
+      </section>
+      <div class="social">
+        <ul class="social-links">
+          <li><img src="./Assets/github.svg" alt="Github symbol" /></li>
+          <li><img src="./Assets/linkedin.svg" alt="Linkedin symbol" /></li>
+          <li><img src="./Assets/angelist.svg" alt="Angelist sysmbol" /></li>
+          <li>
+            <img src="./Assets/Twitter.svg" alt="Twitter sysmbol" />
+          </li>
+          <li>
+            <img src="./Assets/medium.svg" alt="" />
+          </li>
+        </ul>
+      </div>
+      <figure class="background-one">
+        <img src="./Assets/Group.svg" alt="gemoetric design" />
+      </figure>
```

Comment:
- [ ] Kindly add the background decorations as background with CSS, as the image is not part of the content (aesthetic reasons). [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

Reply:
solved